src/bertconf_bert_loader.erl

```diff
@@ -16,27 +16,27 @@ init([]) ->
     process_flag(trap_exit, true),
     ?MODULE = ets:new(?MODULE, [named_table, set, public, {keypos, #tab.name}, {read_concurrency, true}]),
     {Old, Changes} = reload_bert_sync([]),
-    {ok,
-     #state{watches=[begin
-                         {ok,Pid} = dirwatch:start(self(), P, reload_delay()),
-                         Pid
-                     end || P <- find_dirs()],
-            changes=Changes,
-            old_tables=Old},
-     hibernate}.
+    Timer = erlang:start_timer(reload_delay(), self(), reload),
```

Comment:
Mostly because it's easier to start a periodic timer that acts independently of dirwatch then it is to manage the two in concert. The amount of work on wakeup is small enough and the periods high enough that I don't see an immediate need to complicate the logic.

There might be an alternative that I missed and I'm open to suggestions.